bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gems/CanopusFeature.java

```diff
@@ -4,70 +4,31 @@
 import edu.gemini.shared.util.immutable.Some;
 import edu.gemini.shared.util.immutable.None;
 import edu.gemini.spModel.gemini.gems.Canopus;
-import edu.gemini.spModel.guide.GuideProbe;
 import edu.gemini.spModel.obs.context.ObsContext;
 import edu.gemini.spModel.obscomp.SPInstObsComp;
-import edu.gemini.spModel.target.SPTarget;
 import edu.gemini.spModel.target.WatchablePos;
-import edu.gemini.spModel.target.env.GuideProbeTargets;
-import edu.gemini.spModel.target.env.TargetEnvironment;
 import edu.gemini.spModel.target.env.TargetEnvironmentDiff;
 import jsky.app.ot.tpe.*;
-import jsky.app.ot.tpe.feat.TpeGuidePosCreatableItem;
-import jsky.app.ot.util.BasicPropertyList;
-import jsky.app.ot.util.OtColor;
 import jsky.app.ot.util.PropertyWatcher;
 
-import javax.swing.*;
 import java.awt.*;
 import java.awt.geom.AffineTransform;
 import java.awt.geom.Area;
 import java.awt.geom.Point2D;
-import java.awt.geom.Rectangle2D;
-import java.awt.image.BufferedImage;
 import java.beans.PropertyChangeListener;
 import java.util.Collection;
 import java.util.Collections;
 
 /**
  * Draws the Canopus AO field of view and probe ranges.
  */
-public final class CanopusFeature extends TpeImageFeature implements PropertyWatcher, TpeModeSensitive, TpeDragSensitive {
+public final class CanopusFeature extends TpeImageFeature implements PropertyWatcher {
```

Comment:
We no longer need to be `TpeModeSensitive` or `TpeDragSensitive` since we are no longer drawing the CWFS1 and CWFS2 ranges inside the CWFS patrol field.